hrpsys_ros_bridge/test/hrpsys-samples/README.md

```diff
@@ -67,8 +67,8 @@ AutoBalancer is hrpsys-base RTC to generate walking pattern and control Center O
 1. Launch hrpsys-simulator
 
     ```
-    export BUSH_CUSTOMIZER_CONFIG_PATH=`rospack find openhrp3`/../OpenHRP-3.1/customizer/sample1_bush_customizer_param.conf
-    rtmlaunch hrpsys_ros_bridge samplerobot.launch hrpsys_precreate_rtc:=PDcontroller USE_UNSTABLE_RTC:=true PROJECT_FILE:=`rospack find hrpsys`/share/hrpsys/samples/SampleRobot/SampleRobot.500.torque.xml
+    export BUSH_CUSTOMIZER_CONFIG_PATH=$(catkin locate --devel)/share/OpenHRP-3.1/customizer/sample1_bush_customizer_param.conf
+    rtmlaunch hrpsys_ros_bridge samplerobot.launch hrpsys_precreate_rtc:=PDcontroller USE_UNSTABLE_RTC:=true PROJECT_FILE:=$(catkin locate --devel)/share/hrpsys/samples/SampleRobot/SampleRobot.500.torque.xml
```

Comment:
`catkin locate --devel` works only when current directory is inside catkin workspace.
In addition, if you install openhrp3 or hrpsys by apt, those commands have no chance to work.
How about writing as follows (please confirm as I might make typo):
```
    If you install openhrp3 from apt (`roscd openhrp3` takes you under `/opt/ros`):
    ```
    export BUSH_CUSTOMIZER_CONFIG_PATH=`rospack find openhrp3`/../OpenHRP-3.1/customizer/sample1_bush_customizer_param.conf
    ```
    Otherwise:
    ```
    roscd openhrp3
    export BUSH_CUSTOMIZER_CONFIG_PATH=$(catkin locate --devel)/share/OpenHRP-3.1/customizer/sample1_bush_customizer_param.conf
    ```
    If you install hrpsys from apt (`roscd hrpsys` takes you under `/opt/ros`)
    ```
    rtmlaunch hrpsys_ros_bridge samplerobot.launch hrpsys_precreate_rtc:=PDcontroller USE_UNSTABLE_RTC:=true PROJECT_FILE:=`rospack find hrpsys`/share/hrpsys/samples/SampleRobot/SampleRobot.500.torque.xml
    ```
    Otherwise:
    ```
    roscd hrpsys
    rtmlaunch hrpsys_ros_bridge samplerobot.launch hrpsys_precreate_rtc:=PDcontroller USE_UNSTABLE_RTC:=true PROJECT_FILE:=$(catkin locate --devel)/share/hrpsys/samples/SampleRobot/SampleRobot.500.torque.xml
    ```
```